docs/user-guides/forwarding-logs-via-http/deploy/filter-grep.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: logging.kubesphere.io/v1alpha2
+kind: Filter
+metadata:
+  name: grep
+  labels:
+    http-output-demo: "true"
+spec:
+  match: "*"
+  filters:
+  - grep:
+      regex: "log  ERRO"
```

Comment:
It's more accurate to use regex like below:
log [ ]+ERROR[ ]+

Reply:
Yep, I change it to "log \sERRO\s"

Note that the value format is `KEY REGEX`. Only one space in between.